crates/turborepo-auth/src/auth/logout.rs

```diff
@@ -1,5 +1,34 @@
-use turborepo_ui::{cprintln, GREY, UI};
+use std::fs;
 
-pub fn logout(ui: &UI) {
-    cprintln!(ui, GREY, ">>> Logged out");
+use tracing::error;
+use turborepo_api_client::Client;
+use turborepo_ui::{cprintln, GREY};
+
+use crate::{Error, LogoutOptions};
+
+pub fn logout<T: Client>(options: &LogoutOptions<T>) -> Result<(), Error> {
+    if let Err(err) = remove_token(options) {
+        error!("could not logout. Something went wrong: {}", err);
+        return Err(err);
+    }
+
+    cprintln!(options.ui, GREY, ">>> Logged out");
+    Ok(())
+}
+
+fn remove_token<T: Client>(options: &LogoutOptions<T>) -> Result<(), Error> {
+    let global_config_path = options.path;
+    let content =
+        fs::read_to_string(global_config_path).map_err(|e| Error::FailedToReadAuthFilePath {
+            path: global_config_path.clone(),
+            error: e,
+        })?;
```

Comment:
I would suggest using just `global_config_path.read_to_string` as we use `fs_err` which automatically includes the path in the io errors.
```suggestion
    let content = global_config_path.read_to_string()?;
```